wicketd/src/update_tracker.rs

```diff
@@ -830,6 +830,23 @@ impl Drop for SetTrueOnDrop {
     }
 }
 
+fn model_is_cosmo(model: &str) -> bool {
+    model == "913-0000023"
```

Comment:
Is this the way we should identify gimlet vs cosmo? (Both here and in Nexus.) If so, should we have something like `InternalHostType::from_model(model: &str) -> Result<Self, ...>` that matches on specific  models? (Does gimlet have exactly one model string?)

I guess an alternative is to parse the `gimet-` or `cosmo-` prefix from the SP `BORD`, but that's similarly fallible. :-/